mlflow/store/gcs_artifact_repo.py

```diff
@@ -6,6 +6,7 @@
 from mlflow.entities import FileInfo
 from mlflow.store.artifact_repo import ArtifactRepository
 from mlflow.utils.file_utils import relative_path_to_artifact_path
+from google.auth.exceptions import DefaultCredentialsError
```

Comment:
I realized that this might actually be your test failure problem: we didn't want "google" imports at the top level because users might want to run MLflow without bringing in the whole Google client library, so we put all "google" imports in the actual methods below. Can you move this import into the `_get_bucket` method where it uses `DefaultCredentialsError` instead?

Reply:
(In particular, the failing tests were ones that build models for serving, where the environment doesn't contain the Google package by default; the top level tests do have that package in the test environment.)

Reply:
Ah that makes sense! Ok good to know - I've updated it now 👍